packages/storybook/sticky-navigation.stories.js

```diff
@@ -0,0 +1,228 @@
+import { html } from 'lit-html';
+import { useEffect } from '@storybook/client-api';
+import '@conversionxl/cxl-ui/src/components/cxl-marketing-nav.js';
+import contextMenuItems from './navigation.data.json';
+
+export default {
+  title: 'CXL Navigation - Sticky'
+};
+
+export const CxlMarketingNav = () => {
+  useEffect(() => {
+    // Populate `<cxl-marketing-nav>` context menus.
+    const cxlMarketingNavElement = document.querySelector('cxl-marketing-nav');
+
+    cxlMarketingNavElement.contextMenuItems = contextMenuItems;
+  }, []);
+
+  return html`
+    <header class="site-header">
```

Comment:
Why do we sticky an outside container instead of `cxl-marketing-nav` element itself?

Reply:
We don't necessarily have to, I tried to keep it separate from **cxl-marketing-nav" so I have a clear understanding of sideeffects that they are not because of custom elements if any, so doesn't have false positive, we can definitely utilize once we have useable solution.